src/H5private.h

```diff
@@ -42,21 +47,18 @@
 #endif /* H5_HAVE_THREADSAFE */
 
 /*
- * Include ANSI-C header files.
+ * Include ANSI-C header files that aren't included in H5public.h
  */
-#ifdef H5_STDC_HEADERS
```

Comment:
lol

Reply:
Have you done a search for the H5_STDC_HEADERS? I seem to recall tripping over it somewhere else.

Reply:
I'll look again, but I don't think this is used anywhere else.

Reply:
Update: I yanked it from the perform code.